src/services/api/base.ts

```diff
@@ -0,0 +1,55 @@
+import { request, RequestOptions } from 'https';
+
+import { ITokenProvider } from '../token/provider';
+import { IncomingMessage } from 'http';
+
+export type Method = 'GET' | 'HEAD' | 'POST' | 'PUT' | 'DELETE' | 'CONNECT' | 'OPTIONS' | 'TRACE' | 'PATCH';
+
+export class ApiClientBase {
+
+  constructor(
+    private baseUrl: string,
+    private tokenProvider: ITokenProvider,
+  ) {
+  }
+
+  protected async request<T>(method: Method, path: string, refreshToken: boolean = false): Promise<T> {
+    const url = this.baseUrl + path;
+    const token = await ((
+       refreshToken ? this.tokenProvider.refresh() : this.tokenProvider.get()
+    ));
```

Comment:
```suggestion
    const token = await (
       refreshToken ? this.tokenProvider.refresh() : this.tokenProvider.get()
    );
```
カッコが多いっす
フォーマッタ入れてたらこういうことにはならんのでおすすめ

Reply:
死にます………
次の PR で ESLint + Prettier 導入します……